Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -148,6 +148,16 @@ - (void)beginLocationUpdates
   [_locationManager startUpdatingLocation];
 }
 
+- (BOOL) isNewLocation:(CLLocation*) location
+{
+  if ([_lastLocationEvent objectForKey:@"coords"] != [NSNull null]){
```

Comment:
Also, please use [ ] access syntax for dictionary, i.e.

    if (_lastLocationEvent[@"coords"] != (id)kCFNull) {